membership-common/src/main/scala/com/gu/zuora/ZuoraSoapService.scala

```diff
@@ -31,13 +34,26 @@ object ZuoraSoapService {
   }
 }
 
-class ZuoraSoapService(soapClient: soap.ClientWithFeatureSupplier)(implicit ec: ExecutionContext) extends SafeLogging {
+trait SoapClient[M[_]] {
+
+  def getAccountIds(contactId: ContactId)(implicit logPrefix: LogPrefix): M[List[AccountId]]
+}
```

Comment:
had to add the trait (with the type parameter) because just the one method was mocked in a test